autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -196,3 +213,16 @@ def _add_models(models, options, name_prefix, model_fn):
     else:
         models.append(model_fn(name_prefix, options))
 
+
+def __is_fastai_available():
+    try:
+        with warning_filter():
+            import fastai
+            import torch
+        fastai_imported = True
+    except ImportError:
+        fastai_imported = False
+
+    if not fastai_imported:
+        warnings.warn('fastAI models cannot be created because fastai/torch are not installed.')
```

Comment:
Can you add install instructions at the end of warning message? e.g. "pip install fastai" (I believe this'll install torch as well).

Reply:
added